pkg/trace/api/info_test.go

```diff
@@ -187,6 +188,7 @@ func TestInfoHandler(t *testing.T) {
 		"/v0.6/stats",
 		"/v0.1/pipeline_stats",
 		"/appsec/proxy/",
+		"/evpIntakeProxy/v1/",
```

Comment:
Isn't `evp-intake-proxy` or `evp_intake_proxy` better? The former seems more friendly to general path naming style and the latter more in line with other things we already have (e.g. `pipeline_stats`).

Also, other similar paths have the `/input` suffix. Do we need that too?